packages/cxl-ui/src/index-vaadin-grid.js

```diff
@@ -0,0 +1,3 @@
+import * as CXLVaadinGrid from '../../cxl-vaadin-grid/src/index.js';
+
+export { CXLVaadinGrid };
```

Comment:
Hm, I think we should just `import @vaadin/vaadin-grid;` here and that is all?

Why do we need `packages/cxl-vaadin-grid` separately set up?

Reply:
Its more of an organization point of view, I did it like this. It can be adapted for custom js, in case we want to split it. Unless you see issue with this approach, I like to keep it.

Reply:
I'll remoe it to keep things clearer in v1. Can bring it back after more people identify the need.